api/v1/infinispan_webhook.go

```diff
@@ -123,6 +123,17 @@ func (i *Infinispan) Default() {
 				i.Spec.Service.Sites.Locations[idx].URL = fmt.Sprintf("%s://%s:%d", consts.StaticCrossSiteUriSchema, *location.Host, port)
 			}
 		}
+
+		if i.Spec.Service.Sites.Local.Discovery == nil {
+			i.Spec.Service.Sites.Local.Discovery = &DiscoverySiteSpec{}
+		}
+		if i.Spec.Service.Sites.Local.Discovery.Type == "" {
+			i.Spec.Service.Sites.Local.Discovery.Type = GossipRouterType
+		}
+		if i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter == nil {
+			i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter = new(bool)
+			*i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter = true
```

Comment:
FYI you can do this as a one-liner with `i.Spec.Service.Sites.Local.Discovery.LaunchGossipRouter = pointer.BoolPtr(true)`